src/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixer.php

```diff
@@ -209,6 +235,10 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
+            (new FixerOptionBuilder('allow_future_params', 'Whether `param` annotation for future method signature are allowed.'))
```

Comment:
Is `allow_future_params` actually a correct name? I wonder if it's always about the "future" 🤔. Maybe it should be `allow_hidden_params` or something like that?

Reply:
The only live use-case I'm aware of is coming from Symfony and they use it only for future-params.
I am not familiar with other usage of this approach and what "hidden" and "NON-future" params we could have.

happy to learn more, if there are such cases in wild.

Reply:
IMO "hidden" is more suitable because it does not assume a specific reason as to why the parameter is commented out, even if we only know one possible reason for now. But I'm fine with keeping "future" if you prefer that.

Reply:
I do prefer `allow_hidden_params` too. The option only changes the behavior of the rule regarding commented out parameters. The reason why the parameters are commented out in the first place is not relevant and we don't need to learn about more use cases IMO, as long as we decide to support at least one. `allow_hidden_params` is also more explicit.

Reply:
> The reason why the parameters are commented out in the first place is not relevant and we don't need to learn about more use cases IMO (...). `allow_hidden_params` is also more explicit.

Exactly - this is an "implementation detail", and the essence of the option is handling params that are "hidden" (commented out). Moreover, `allow_hidden_params` name covers currently known use case well while being open for any other use case related to commented out params. Using `allow_future_params` not only is less explicit about what it does, but also most probably would not fit well for any other use case. So it's better to use more generic option instead of deprecating it later in favour of the new one.

Reply:
applied